db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,7 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
-      t.string :link
+      t.string :simplified_name
       
```

Comment:
yes i forgot about this. can I do this after the merge? it's not so mission critical...